ChatRPG/Pages/CampaignPage.razor.cs

```diff
@@ -48,9 +69,15 @@ protected override async Task OnInitializedAsync()
 
         _campaign = await PersisterService!.LoadFromCampaignIdAsync(
             CampaignMediatorService!.UserCampaignDict[_loggedInUsername!]);
+        _npcList = _campaign.Characters.ToList();
+        _currentLocation = _campaign.Environments.LastOrDefault();
+        _mainCharacter = _campaign.Player;
         if (_campaign != null)
```

Comment:
```suggestion
        if (_campaign)
```

Reply:
Hold the phone, det er sabotage! 🤣

Reply:
Men helt skidt var det ikke for det fik mig til at indse at de tre tilføjede linjer (72, 73, og 74) nok burde være inde i null checket! Så tak 😄 Nu må du gerne re-review!